### PR TITLE
added Dockerfile

### PR DESCRIPTION
**Pull request by leannert**

### CODE REVIEW

File: assignments/TONG/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Followingly, we build and tag the container image, using the code format below (in the terminal):
+```
+podman build -t [image-name] -f [path/to/containerfile]
+```
+
+## Running the Image
+After building the container image, we are able to run the image with the code format below:
+```
+podman run -it --rm [image-name]
```
Comment (sallyom) [inline assignments/TONG/README.md:44]: The `-it` in the `podman run` command isn't needed for this image to run.
Run `podman run --help` to see the info for the `-i --interactive` and `-t --tty`, they are necessary when  you need to interact with a running container (like playing tic tak to or if you want to use shell commands while the container is running)

**Review by sallyom — CHANGES_REQUESTED** [review]
Nice documentation! I left a comment, also, do you want to include the `Notes` file? We can go over these in class 4/25 :)

File: assignments/TONG/README.md (hunk 1 above)
Comment (leannert) [reply assignments/TONG/README.md]: I see! Thank you for letting me know :)

Comment (leannert) [reply assignments/TONG/README.md]: As a response to the comment (change request) below, I do want to keep the `Notes` file for future use. I made these while I was figuring out how to run my image, and I found them very helpful.

**Comment by sallyom** [comment]
LGTM!

**Review by sallyom — APPROVED** [review]
Nice work

**Pull request merged**